### PR TITLE
Embed the decision tree as modal

### LINKED ISSUE #165: Embed the decision tree opens as a modal
- [x] [NLDS does not have a modal](https://nldesignsystem.nl/modal-dialog/#implementaties), so we need to create this component.
- [x] Depends on #147, designs are there.
- [ ] If you have completed the decision tree auto fill the result on the 'create project page'

Technical notes
- we may need to alter the decision tree to accept a callback hook to handle output

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# Description

- Adds modal
- Adds functionality to check radiobuttons and checkboxes according to the outcome of the decision tree upon closing the modal.

There is currently one problem. In the form under the category "Role" a user can select two options "aanbieder" or "gebruiksverantwoordelijke", because both can be selected at the same time. The problem occurs in the following situation. Image a user uses the decsion tree and for example "aanbieder" is set as role. If the user again uses the decsion tree and now the result is "gebrruiksverantwoordelijke" then "aanbieder' is still set. I don't know how to "uncheck" a checkbox, because there does not seem to be a "checked" indication in the HTML when a checkbox is checked.

Resolves #165 

## Checklist

Please check all the boxes that apply to this pull request using "x":

-   [x] I have tested the changes locally and verified that they work as expected.
-   [x] I have followed the project's coding conventions and style guidelines.
-   [x] I have rebased my branch onto the latest commit of the main branch.
-   [x] I have squashed or reorganized my commits into logical units.
-   [x] I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

Screenshot 2024-10-07 at 10 17 32

### CODE REVIEW

File: amt/site/templates/projects/new.html.j2 [hunk 1]
```diff
@@ -34,12 +34,16 @@
...
-                        <a class="rvo-link rvo-link--normal"
-                           href="https://ai-act-decisiontree.apps.digilab.network/">{% trans %}The AI Act decision tree {% endtrans %}</a>
-                        {% trans %}helps you determine the value of these fields.{% endtrans %}
+                        {% trans %}The AI Act profile provides insight into, among other things, the type of AI system and the associated obligations from the European AI Act. If you already know the type of AI system you can fill in the below fields. Otherwise, you can find your AI Act Profile with the AI Act Decision tree.{% endtrans %}
+
                     </p>
                     <div class="utrecht-form-fieldset rvo-form-fieldset">
+                    <p class="utrecht-button-group">
+                        <button id="decision-tree-modal-button" class="utrecht-button utrecht-button--secondary-action utrecht-button--rvo-md utrecht-button-group__align-right" onclick="amt.openModal()" type="button">
+                        Find your AI Act profile
```
Comment (laurensWe) [inline amt/site/templates/projects/new.html.j2:43]: This should also be embraced in { %trans%} blocks right?
```suggestion
                        { %trans%} Find your AI Act profile{ %endtrans%} 
```

File: amt/site/templates/projects/new.html.j2 [hunk 2]
```diff
@@ -34,12 +34,16 @@
...
                     <p class="rvo-text rvo-text--no-margins">
-                        {% trans %}The AI Act profile provides insight into, among other things, the type of AI system and the associated obligations from the European AI Act.{% endtrans %}
-                        <a class="rvo-link rvo-link--normal"
-                           href="https://ai-act-decisiontree.apps.digilab.network/">{% trans %}The AI Act decision tree {% endtrans %}</a>
-                        {% trans %}helps you determine the value of these fields.{% endtrans %}
+                        {% trans %}The AI Act profile provides insight into, among other things, the type of AI system and the associated obligations from the European AI Act. If you already know the type of AI system you can fill in the below fields. Otherwise, you can find your AI Act Profile with the AI Act Decision tree.{% endtrans %}
+
                     </p>
                     <div class="utrecht-form-fieldset rvo-form-fieldset">
+                    <p class="utrecht-button-group">
```
Comment (laurensWe) [inline amt/site/templates/projects/new.html.j2:41]:
```suggestion

```

Comment (laurensWe) [reply amt/site/templates/projects/new.html.j2:41]: Is the paragraph really needed if you just have 1 button?

Comment (ChristopherSpelt) [reply amt/site/templates/projects/new.html.j2:41]: It is needed for right alignment.

**Comment by anneschuth** [comment]
Nice! LGTM!

**Review by anneschuth — DISMISSED** [review]

**Review by laurensWe — DISMISSED** [review]
LGTM :)

**Review by laurensWe — APPROVED** [review]
LGTM

**Pull request merged**